Code/ForceField/MMFF/AngleBend.cpp

```diff
@@ -1,4 +1,5 @@
 // $Id$
+//  Copyright (C) 2014-2025 Greg Landrum and other RDKit contributors
```

Comment:
```suggestion
//  Copyright (C) 2014-2025 Paolo Tosco and other RDKit contributors
```
and remove the other copyright lines below

this applies to the other files as well

Reply:
Done, thanks